pkg/blockdevice/lvm/volume_groups.go

```diff
@@ -118,6 +158,15 @@ func (vg *VolumeGroup) Change(ctx context.Context, rawArgs string) (bool, error)
 }
 
 func (vg *VolumeGroup) LockStart(ctx context.Context, rawArgs string) (bool, error) {
+	exists, err := vg.WaitForAppearance(ctx)
```

Comment:
This change improves lvmlockd interaction because we wait until the VG/LVs appear on the node before attempting to open the lock space.